zetaclient/orchestrator/bootstrap.go

```diff
@@ -481,6 +478,20 @@ func syncObserverMap(
 	return added, removed, nil
 }
 
+func btcDatabaseFileName(chain chains.Chain) string {
+	// btcDatabaseFilename is the Bitcoin database file name now used in mainnet and testnet3
+	// so we keep using it here for backward compatibility
+	const btcDatabaseFilename = "btc_chain_client"
+
+	// For additional bitcoin networks, we use the chain name as the database file name
+	switch chain.ChainId {
+	case chains.BitcoinMainnet.ChainId, chains.BitcoinTestnet.ChainId:
```

Comment:
Let's do the following: 

If chain is in `[mainnet, testnet3]`, then name the db as `btc_chain_client`. Otherwise `fmt.Sprintf("btc_chain_client_%d", chainID)`

Reply:
The testnet is now running on database name `chain.Name`. This might involve some operation work (testnet only) to manually set the last scanned height on restart. I can change it to the pattern if @gartnera feels okay.

Reply:
That pattern is fine I think

Reply:
Okay. Let's do it.

Reply:
https://github.com/zeta-chain/node/pull/3225/commits/54b452f77d31b8079aa32dc3d64793c970104329